### PR TITLE
Adding UD models 1.0 to the website.

### PR DESCRIPTION
**Pull request by jzonthemtn**
Adding UD models 1.0 to the website.

### CODE REVIEW

File: src/main/jbake/content/news/news-2021-05-30.ad [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+2021-05-30
+:jbake-type: post
+:jbake-tags: news
+:jbake-status: published
+:category: news
+:idprefix:
+
+== OpenNLP Pre-trained Models Available
+
+Pre-trained sentence, parts of speech, and token models are now available for English, French, Italian, German, and Dutch. These models were trained on the Universal Dependency corpus and are intended to provide usable models under the Apache 2.0 license. See the models README for more information on the models including how each was created and evaluated.
```
Comment (kinow) [inline src/main/jbake/content/news/news-2021-05-30.ad]: Missing newline, but not super important

File: src/main/jbake/content/models.ad [hunk 2]
```diff
@@ -70,3 +213,11 @@ That can easily be done with:
 
 More information about release signing and verifying signatures can
 be found https://www.apache.org/dev/release-signing.html[here].
+
+# SourceForge Models
+
+The models on Sourceforge for 1.5.0 are http://opennlp.sourceforge.net/models-1.5/[found here]
```
Comment (kinow) [inline src/main/jbake/content/models.ad:219]: Maybe use https link (nit picking - IDE linter is now detecting http links as low issues, not important)

**Review by kinow — APPROVED** [review]

File: src/main/jbake/content/models.ad (hunk 2 above)
Comment (jzonthemtn) [reply src/main/jbake/content/models.ad:219]: Looks like https://opennlp.sourceforge.net/models-1.5/ doesn't work. Do you know if there's a different https address for sourceforge?

Comment (kinow) [reply src/main/jbake/content/models.ad:219]: Oh, you are right. I think they don't offer an HTTPS version at all. Sorry!

**Pull request merged**